x/protocolpool/keeper/keeper.go

```diff
@@ -27,9 +27,14 @@ type Keeper struct {
 
 	authority string
 
+	// Global variables
+	toDistribute uint64
```

Comment:
will this carry over with starting and stopping a node?

Reply:
Now `toDistribute` is a package level scope var and so this var will exist as long as the module or application is running. 
Therefore, I think toDistribute variable will maintain its value. But if the application is restarted, the variable will be reinitialized, losing its previous value.